CHANGELOG.md

```diff
@@ -1,5 +1,55 @@
 # PowerShell Extension Release History
 
```

Comment:
v2021.2.0
Monday, February 22, 2021
📺✨ New semantic highlighting. (Thanks @justinytchen!)
🐢 ✨ Added Pester v5 support to problem matcher. (Thanks @fflaten!)
✨👮 Updated PSScriptAnalyzer to 1.19.1.
- Fixes formatting bugs! (Thanks @bergmeister!)
🐛 Many squashed bugs

Reply:
Thank you!